Gemfile

```diff
@@ -1,3 +1,6 @@
 source 'https://rubygems.org'
 
 gemspec
+# Only for development
+gem 'aliquot', git: 'https://github.com/Verseth/aliquot.git', branch: 'allow-ruby31'
```

Comment:
This is only temporary. It had to be changed because `aliqout` doesn't allow Ruby higher than 3.0 (so bundler would cause the tests to fail)